sheet/ClassSkill.qml

```diff
@@ -6,7 +6,12 @@ Skill {
     modifiers: [
         Bonus {
             name: "Class Skill"
-            amount: 3
+            amount: if(ranks>=1){
```

Comment:
You can use ternary operator to replace the if expression.

A ternary expression for this would look like: (ranks >= 1) ? 3 : 0.